scanner/matcher/matcher.go

```diff
@@ -114,8 +114,7 @@ func NewMatcher(ctx context.Context, cfg config.MatcherConfig) (Matcher, error)
 	return &matcherImpl{
 		libVuln:       libVuln,
 		metadataStore: metadataStore,
-
-		updater: u,
+		updater:       u,
```

Comment:
Just want to confirm:
The `updater` here is going to fetch data from `cfg.DefinitionsURL` in a interval, right?

Reply:
Yes, the updater will fetch at every defined interval, see https://github.com/stackrox/stackrox/blob/b67133ee4d2de45f6f6da514d136ffb3bb83c692/scanner/matcher/updater/updater.go#L206